Tests/MusicXMLTests/ScoreTests/Reve/manifest.yaml

```diff
@@ -1,9 +1,9 @@
 parsing:
     xml_file_name: reve.musicxml
-    assert: false
+    assert: true
 
 comparison:
```

Comment:
I recommend commenting out these three lines, so it's easier to reenable them later.